src/docs/vscode-ext/reference/command-palette.md

```diff
@@ -0,0 +1,35 @@
+<style>
+    .md-typeset table:not([class]) {
+        background-color: var(--md-primary-fg-color);
+    }
+</style>
+
+# Command palette
+
+Truffle for VSCode provides several commands for interacting with ledgers, and building, compiling,
+and testing your smart contracts.
+These features are enabled through the command palette and several "right-click" shortcut menus.
```

Comment:
Both "right-click" and "right click" are used. WSG uses "right-click" as the action.